docker/build_scripts/build_utils.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
+# Helper utilities for build
+
+PYTHON_DOWNLOAD_URL=https://www.python.org/ftp/python
+OPENSSL_DOWNLOAD_URL=http://www.openssl.org/source
+GET_PIP_URL=https://bootstrap.pypa.io/get-pip.py
+
+
+function check_var {
+    if [ -z "$1" ]; then
+        echo "required variable not defined"
+        exit 1
+    fi
+}
+
+
+function pyver_ge {
+    # Echo 1 if first python version is greater or equal to second
```

Comment:
Probably not the easiest way, no -- over the versions that we care about, lexicographic order is actually fine... (Guido has said that there will never be a 3.10; it will be called 4.0 instead. 2.7.10 was the first python release to ever have a >1 digit number, and even then it's still restricted to the micro field which we don't care about here).